tamr_client/instance.py

```diff
@@ -10,3 +10,20 @@ def origin(instance: Instance) -> str:
         return f"{instance.protocol}://{instance.host}"
     else:
         return f"{instance.protocol}://{instance.host}:{instance.port}"
+
+
+def version(session: Session, instance: Instance) -> str:
+    """Return the Tamr version for an instance.
+
+    Args:
+        session: Tamr Session
+        instance: Tamr instance
+
+    Returns: Version
+
+    """
+    return _version(session, instance)["version"]
+
+
+def _version(session: Session, instance: Instance) -> dict:
```

Comment:
I'm not sure this needs to be separated into these two functions.  I think I would prefer it to be one.

Reply:
I mean, it doesn't NEED to be, but there's more in the response than just the version number itself. Perhaps that would be useful to someone, perhaps it wouldn't, but that's why it's not exposed. I feel like the only argument against not building it this way is that it requires building 1 more function, but it's already built. I would prefer it to be two.

Reply:
I changed my mind. None of that extra stuff should be accessible in any way.